modules/installation-network-user-infra.adoc

```diff
@@ -183,6 +183,12 @@ the Cluster Version Operator on port `9099`.
 |N/A
 |IPsec Encapsulating Security Payload (ESP)
 
+|NTP
```

Comment:
This column represents "Protocol", so I think it should be changed to UDP.

Reply:
@sr1kar99 It is OK from my side to write UDP under "Protocol" column.

Reply:
@jinyunma @dushyantu2 
Updated as per the suggestion.
Thanks!

Reply:
Since this now says "UDP" instead of "NTP", should this port be added up here with the other UDP ports?

![image](https://github.com/openshift/openshift-docs/assets/61474374/f5448680-d946-435f-a95e-d3d7816146b6)

Reply:
Yes! It makes sense to add it there.